.github/workflows/smoke-test-daily.yml

```diff
@@ -44,36 +46,24 @@ jobs:
 
             - name: Run 'Update WooCommerce' test.
               working-directory: plugins/woocommerce
-              id: e2e-update
               env:
                   UPDATE_WC: true
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js update-woocommerce.spec.js
+              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js update-woocommerce.spec.js
 
             - name: Run the rest of E2E tests.
               timeout-minutes: 60
               working-directory: plugins/woocommerce
-              id: e2e
               env:
-                  E2E_MAX_FAILURES: 15
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js basic.spec.js
+                  E2E_MAX_FAILURES: 25
+              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js
 
             - name: Generate Playwright E2E Test report.
-              id: generate_e2e_report
-              if: |
-                  always() &&
-                  (
-                    steps.e2e-update.conclusion != 'cancelled' ||
-                    steps.e2e-update.conclusion != 'skipped' ||
-                    steps.e2e.conclusion != 'cancelled' ||
-                    steps.e2e.conclusion != 'skipped' 
-                  )
+              if: success() || failure()
```

Comment:
I realized it's better to be careful in using `always()` because it will cause the job to continue running even when we try to cancel the run.

Taken from https://docs.github.com/en/actions/learn-github-actions/expressions#always
> ### always
> 
> Causes the step to always execute, and returns true, even when canceled